src/Host/Client/Impl/Host/RHostConnector.cs

```diff
@@ -100,13 +106,12 @@ internal abstract class RHostConnector : IRHostConnector {
             }
         }
 
-        public async Task<RHost> ConnectAsync(string name, IRCallbacks callbacks, string rCommandLineArguments = null, int timeout = 3000, CancellationToken cancellationToken = new CancellationToken()) {
+        public async Task<RHost> ConnectAsync(string name, IRCallbacks callbacks, int timeout = 3000, CancellationToken cancellationToken = new CancellationToken()) {
```

Comment:
The UI can stay for now. I just think the lower level API should be more flexible in that regard, given that this flexibility is inherent in the design of other components.

Thinking of possible future scenarios - some of the helper sessions might not want to apply the user-specified command line at all (in fact, I'm not sure that the ones that we currently have actually need them), while others might specify some arguments of their own.

---
In reply to: [75949601](https://github.com/Microsoft/RTVS/pull/2192#discussion_r75949601) [](ancestors = 75949601,75946453)